ChangeLog

```diff
@@ -1,3 +1,79 @@
+* Tue Apr 28 2020 Klaus Wenninger <klaus.wenninger@aon.at> Pacemaker-2.0.4
+- Changesets: 442
+- Diff: 269 files changed, 21845 insertions(+), 13363 deletions(-)
+
+- Features added since Pacemaker-2.0.3
+  + build: Add support for glib-based unit tests.
+  + build: bumplibs: add git coloring, show PRs merged & show commits
+                     touching each library
+  + controller: add new feature 'shutdown-locks'
+        Optionally allow a gracefully shutdown node to have the resources
+        locked to it for a configurable time.
+        So that it can be restarted with exactly the same resources
+        running as before.
+  + controller/fencer/scheduler: add new feature 'priority-fencing-delay'
+        Optionally derive the priority of a node from the resource-priorities
+        of the resources it is running.
+        In a fencing-race the node with the highest priority has a certain
+        advantage over the others as fencing requests for that node are
+        executed with an additional delay.
+        controlled via cluster option priority-fencing-delay (default = 0)
+  + libcrmcommon: Add a formatter for logs
+  + libpe_status: Add several new functions for working with tags.
```

Comment:
Ditto for the above two

Internal API changes (most obviously functions with a double underbar) definitely don't need to be mentioned. Public API is debatable, maybe we should start listing them under their own heading, but I wouldn't bother for this release

Reply:
Apart from a section dealing with API changes we might as well think of one that lists some highlights for developers.